lib/build.api.js

```diff
@@ -11,7 +11,7 @@ module.exports = function (Aquifer) {
   /**
    * Creates a build object from which operations can be initiated.
    *
-   * @param string destination path to destination folder.
+   * @param {string} destination path to destination folder.
```

Comment:
I see you've made this change for all @param docs in this file. Is this in the JSDOC spec? We have not been doing this consistently, so we should either apply it to the whole codebase, or none at all. I'm not opposed to it, just looking for consistency.